test/source/mock/google/strategies/send-message-strategy.ts

```diff
@@ -473,6 +473,8 @@ export class TestBySubjectStrategyContext {
       this.strategy = new SmimeSignedMessageStrategy();
     } else if (GMAIL_RECOVERY_EMAIL_SUBJECTS.includes(subject)) {
       this.strategy = new SaveMessageInStorageStrategy();
+    } else if (subject.includes('The PWD encrypted message does not include the BCC recipient')) {
```

Comment:
this subject isn't used anymore, let's remove it

Reply:
yes - i'll remove it.